### DISCUSSION TITLE
Is there some kind of regex filtering on spec folder names?

### DISCUSSION THREAD

**Discussion by patrickshaughnessy** [cypress-io/cypress#15083] [Questions and Help] [answered]
Getting a really weird behavior in Cypress 6.4.0 (upgrading from 5):

We have a spec folder at `<root>/cypress/specs/ice/login/auth/<additional nested folders / files>`.

Cypress config is setup fine, no issues except that this `/auth` folder does not appear in the Cypress UI test runner. Changing this to `/auths` or `/something` else makes all the files appear as normal for running. Changing other folder names to `/auth` causes them to disappear as well.

Is there some kind of regex filtering on spec folder names which causes the folder name `auth` to not show up in the UI?

**Comment by bahmutov** [comment, marked as answer]
No, there should not be. Does it find the expected files if you do `cypress run --spec '**auth**'`? Can you fork https://github.com/cypress-io/cypress-test-tiny and add empty spec files to recreate the problem there?

**Reply by patrickshaughnessy** [reply]
Hi @bahmutov Thanks for the response. Tried to reproduce on the test tiny repo, but wasn't able to. This means it must be something in our code base. It's rather large (lots of tests!), but I will check to see if some config is being loaded or overwritten somewhere.
